lib/scripts/repo_export_in_batches.rb

```diff
@@ -4,7 +4,10 @@
 #   ./bin/rails r lib/scripts/repo_export_in_batches.rb > "$(date +'%Y-%m-%d-%H%M%S')_repo_export_in_batches.log"
 # OR
 #   rm -rf tmp/export/; mkdir tmp/export/; ./bin/rails r lib/scripts/repo_export_in_batches.rb > "$(date +'%Y-%m-%d-%H%M%S')_repo_export_in_batches.log"
-#
+
+puts "---------\nRemoving previous export (if it exists) at tmp/export\n---------"
+FileUtils.rm_rf(Dir['tmp/export'])
```

Comment:
I tested this out locally and on staging so I'm confident this will work on production as well.

Reply:
Does this just delete the contents of `/tmp/export` or will the directory need to be created again? Also, can we update the comments/instructions above?

Reply:
Nice catch! For some reason I thought that file writing would just build the path, but I think I'm conflating with something else. Anyways, that's been changed and so have the instructions.